org-formation/060-cloudtrail/_tasks.yaml

```diff
@@ -1,9 +1,25 @@
 Parameters:
   <<: !Include '../_parameters.yaml'
 
+CloudwatchTrailKmsKey:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.4/templates/KMS/kms-key.yaml
+  StackName: !Sub '${resourcePrefix}-cloudwatch-trail-kms-key'
+  Parameters:
+    AliasName: alias/cloudwatch-trail
+    AdminPrincipalArns:
+      - !Sub 'arn:aws:iam::${AWS::AccountId}:root'
+      - !Sub 'arn:aws:iam::${AWS::AccountId}:role/OrganizationAccountAccessRole'
+    UserPrincipalArns:
+      - !Sub 'arn:aws:iam::${AWS::AccountId}:role/OrganizationAccountAccessRole'
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+    Account: '*'
+    Region: !Ref primaryRegion
+
 CloudwatchTrailLog:
   Type: update-stacks
-  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.6/templates/Cloudwatch/cloudwatch-log.yaml
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.4.2/templates/Cloudwatch/cloudwatch-log.yaml
```

Comment:
This template isn't found?

Reply:
it's the template in the dependent PR with a new version tag.  the dependent PR needs to be merged first then tag created then this is using the new tag.